spec/support/saml_auth_helper.rb

```diff
@@ -199,7 +199,7 @@ def link_user_to_identity(user, link, settings)
       user,
       build(:service_provider, issuer: settings.issuer),
     ).link_identity(
-      ial: ial2_requested?(settings) ? true : nil,
+      ial: ial2_requested?(settings) || nil,
```

Comment:
`ial` is an integer column, but as far as I can tell, we've always had this helper assigning `true` or `nil`. It seems kind of odd.